bots/bridge-tester/test/_initialization/initialize.test.ts

```diff
@@ -27,7 +27,7 @@ import {
   DEFAULT_BITCOIN_CORE_WALLET,
 } from "../config";
 
-describe("Initialize parachain state", () => {
+describe.skip("Initialize parachain state", () => {
```

Comment:
Is this `skip` on purpose? If so, please add a comment why. 
If it requires fixing outside of this PR, maybe we can add an issue for it?

Reply:
I'll open an issue for this, it's just that we're not using the bridge-tester, those tests are failing and they're not a priority, see the PR description too

Reply:
> see the PR description too

Ah! Reading comprehension ftw. :D

Reply:
Added here: https://github.com/interlay/bots/issues/20